.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,5 +6,5 @@ contact_links:
     url: https://github.com/meilisearch/documentation/issues/new
     about: For documentation issues, open an issue or a PR in the documentation repository
   - name: Support questions & other
-    url: https://github.com/meilisearch/MeiliSearch/discussions/new
+    url: https://github.com/meilisearch/meilisearch/discussions/new
```

Comment:
```suggestion
    url: https://github.com/meilisearch/Meilisearch/discussions/new
```

Reply:
nope the repo name is meilisearch and not Meilisearch :) I checked with thomas and quentin yersterday :)